src/pyOpenMS/tests/unittests/test000.py

```diff
@@ -368,9 +368,9 @@ def testFineIsotopePatternGenerator():
     water = pyopenms.EmpiricalFormula("H2O")
     mw = methanol + water
     iso_dist = mw.getIsotopeDistribution(pyopenms.FineIsotopePatternGenerator(1e-20, False, False))
-    assert len(iso_dist.getContainer()) == 56 # now 33 ?
+    assert len(iso_dist.getContainer()) == 36
```

Comment:
I wonder why it says now 33? and you get 36...

Reply:
I was hoping it's just a typo in the comment...
Otherwise we are in for some numerical instability issues, which are always fun to debug...
lets wait for CI ...